ts/components/ui/Markdown/index.tsx

```diff
@@ -231,15 +231,18 @@ class Markdown extends React.PureComponent<Props, State> {
       height: htmlBodyHeight
     };
 
+    const isLoadingComplete =
```

Comment:
```suggestion
    const isLoading =
```
Is it a better naming?

Reply:
Yup, changed.